streams/src/main/java/org/apache/kafka/streams/kstream/internals/GroupedStreamAggregateBuilder.java

```diff
@@ -97,11 +97,10 @@ <KR, VR> KTable<KR, VR> build(final NamedInternal functionName,
             parentNode = repartitionNode;
         }
 
-        final StatefulProcessorNode<K, V> statefulProcessorNode =
-            new StatefulProcessorNode<>(
+        final ProcessorGraphNode<K, V> statefulProcessorNode =
+            new ProcessorGraphNode<>(
                 aggFunctionName,
-                new ProcessorParameters<>(aggregateSupplier, aggFunctionName),
-                new String[] {storeFactory.storeName()}
```

Comment:
This is for line 57 above and independent from the PR: just a thought, could we pass in `storeFactory.storeName()` in the future?